src/ray/protobuf/gcs_service.proto

```diff
@@ -27,3 +27,41 @@ service JobInfoGcsService {
   // Mark job as finished to gcs server.
   rpc MarkJobFinished(MarkJobFinishedRequest) returns (MarkJobFinishedReply);
 }
+
+message GetActorInfoRequest {
+  // ID of this actor.
+  bytes actor_id = 1;
+}
+
+message GetActorInfoReply {
+  // Data of actor.
+  ActorTableData actor_table_data = 1;
+}
+
+message RegisterActorInfoRequest {
+  // Data of actor.
+  ActorTableData actor_table_data = 1;
+}
+
+message RegisterActorInfoReply {
+}
+
+message UpdateActorInfoRequest {
+  // ID of this actor.
+  bytes actor_id = 1;
+  // Data of actor.
+  ActorTableData actor_table_data = 2;
+}
+
+message UpdateActorInfoReply {
+}
+
+// Service for actor info access.
+service ActorInfoGcsService {
+  // Get actor specification from GCS Service.
+  rpc GetActorInfo(GetActorInfoRequest) returns (GetActorInfoReply);
+  // Register an actor to GCS Service.
+  rpc RegisterActorInfo(RegisterActorInfoRequest) returns (RegisterActorInfoReply);
+  // Update dynamic states of actor in GCS Service.
```

Comment:
```suggestion
  // Update actor info in GCS Service.
```
Note, we should change this method to `UpdateActorState`, because state is the only field that is allowed to mutate. But this can be done later.

Reply:
fixed